plugins/NBBC/class.nbbc.plugin.php

```diff
@@ -219,7 +219,8 @@ function DoURL($bbcode, $action, $name, $default, $params, $content) {
    public function Format($Result) {
       $Result = str_replace(array('[CODE]', '[/CODE]'), array('[code]', '[/code]'), $Result);
       $Result = $this->NBBC()->Parse($Result);
-      return $Result;
+      $Mixed = html_entity_decode($Result);
+      return Gdn_Format::html($Mixed);
```

Comment:
No biggie, if you add ?w=1 to a GitHub diff it kills all the whitespace. I use it with abandon. :)